### PR TITLE
Osimarkrogers patch 7

### PR DESCRIPTION
**Pull request by osimarkrogers**
Updates to OSIsoft_Cloud_Services, Account_Tenant, and Account_Namespace_1 per discussion with Elizabeth McErlean

### CODE REVIEW

File: Documentation/Management/Account_Tenant.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ uid: AccountTenant
 
 # Tenants
 
-APIs to manage an OCS Customer Tenant Account. Unlike Namespaces, Tenants are not scoped to a region. Account level information, such as Users and Clients, is stored in all supported regions.
+APIs to manage an OCS Customer Tenant Account. Unlike Namespaces, Tenants are not scoped to a region. Account level information, such as Users and Clients, is stored in all supported regions for read-only purposes. Write operations for account management and identity management are only supported through the global base URL. Unsupported requests return a status of ``Method Not Found`` (405). The resolution is to utilize the global base URL (https://dat-b.osisoft.com/) for write operations that are not scoped to a namespace.
```
Comment (chad-at-osisoft) [inline Documentation/Management/Account_Tenant.md]: It should be ``Method Not Allowed`` https://httpstatusdogs.com/img/405.jpg

**Comment by chad-at-osisoft** [comment]
Mark,
I submitted a comment.

Chad

File: Documentation/Management/Account_Tenant.md (hunk 1 above)
Comment (osimarkrogers) [reply Documentation/Management/Account_Tenant.md]: Corrected

Comment (osimarkrogers) [reply Documentation/Management/Account_Tenant.md]: Corrected.

**Pull request merged**